packages/spark-core/components/tabs.js

```diff
@@ -0,0 +1,117 @@
+/* global window */
```

Comment:
I don't think we settled on what we were gonna do about doc blocks. I think we should have them - what do you think?